website/pages/en/tutorials.js

```diff
@@ -36,19 +36,11 @@ const tutorialCardData = [{
 }, {
   title: <translate>Add a Pallet to Your Runtime</translate>,
   text: <translate>Add the Nicks pallet to your Substrate node template.</translate>,
-  difficulty: "medium",
+  difficulty: "easy",
   length: "2",
-  prerequisite: false,
+  prerequisite: true,
   version: "2.0.0",
   href: 'tutorials/add-a-pallet/',
-}, {
-  title: "Upgrade a Chain",
-  text: "Perform a forkless runtime upgrade on a running Substrate network.",
-  difficulty: "medium",
-  length: "2",
-  prerequisite: false,
-  version: "2.0.0",
-  href: 'tutorials/upgrade-a-chain/',
```

Comment:
What is the purpose of this change?

Reply:
Just re-ordering a little bit based in difficulty.